src/subcommand/teleburn.rs

```diff
@@ -10,35 +11,10 @@ pub struct Output {
   ethereum: EthereumTeleburnAddress,
 }
 
-#[derive(Debug, PartialEq)]
```

Comment:
test comment plz ignore

Reply:
Review ACK

This reviews suggested changes are to "plz ignore" and that is what I do with code reviews anyways.